pkg/operator/operator.go

```diff
@@ -223,3 +234,28 @@ func getCABundle(restConfig *rest.Config) (*string, error) {
 	}
 	return ptr.String(base64.StdEncoding.EncodeToString(transportConfig.TLS.CAData)), nil
 }
+
+func getVnetGUID(cfg *auth.Config, subnetID string) (string, error) {
```

Comment:
minor: Maybe document in a comment that this function requires the users to grant certain permissions to the identity that Karpenter is running as, which wouldn't otherwise need to be set?

Reply:
Im thinking we should have a document for all of our helm values in general.  That we update as an artifact for each version?